app/models/external_work.rb

```diff
@@ -44,6 +44,12 @@ def set_url_status
     self.update_attribute(:dead, true) unless url_active?(self.url)
   end
 
+  # Allow encoded characters to display correctly in titles
```

Comment:
I think we'd prefer to take the same approach we're using to resolve this problem on works:

https://github.com/otwcode/otwarchive/blob/2ff9cebf7a92f442e98e8c9874e8b50ca2c7f448/app/models/work.rb#L76-L78

That will let us keep the titles encoded in the database and just change how they're displayed, which means we won't have to worry about cleaning up existing titles in the database.

Reply:
That makes sense! Testing the update first, then will push to this branch.